src/Middleware/Diagnostics/test/UnitTests/StatusCodeMiddlewareTest.cs

```diff
@@ -283,4 +284,38 @@ public async Task Reexecute_WorksAfterUseRoutingWithGlobalRouteBuilder()
         var content = await response.Content.ReadAsStringAsync();
         Assert.Equal("errorPage", content);
     }
+
+    [Fact]
+    public async Task SkipStatusCodePages_SupportsEndpoints()
```

Comment:
Should we add a test that verifies we let people change the feature in the body of the endpoint?

```
endpoints.MapGet("/", [SkipStatusCodePages] (c) =>
{
         c.GetFeature<IStatusCodePagesFeature>().Enabled = false;
});
```

Reply:
Except verify that you can set it to `true`! Although it really would be weird to reenable it after adding the attribute.

Reply:
Seems fair! I missed seeing this before the auto-merge kicked it but I'll address in a follow-up.